sankey_chart/README.md

```diff
@@ -0,0 +1,769 @@
+# Preview
+
+![Screen Shot 2023-03-16 at 12 16 23 PM](https://user-images.githubusercontent.com/124118181/225521638-9898c83d-b45e-4ca8-ba80-9a80d1f60d5b.png)
+
+# Sample Data
+```json
```

Comment:
Just linking to the json file is enough, for example: https://github.com/holistics/custom-chart-library/tree/main/bubble_plot